guide.md

```diff
@@ -70,28 +70,46 @@ CERAMIC_RECON_MODE="true" ceramic daemon --ipfs-api http://localhost:5001
 We now have our Ceramic node running on port 7007. In the following steps, we will deploy our composite onto our node.
 
 ## Configure Your Admin DID
+
 In a new terminal in the same root directory as this repository, we can configure an admin DID our node will use in order to deploy our composites. Let's set an environment variable in order to do so:
 
 ```bash
-nvm use 16
+nvm use 20
```

Comment:
@mzkrasner mentioned composedb commands are not recognized on a terminal when running node v20, but got them working with node v18, should we change this?

Reply:
its working for me using 20